examples/express/README.md

```diff
@@ -0,0 +1,42 @@
+# Hydrogen Express Example
+
+This is a Hydrogen example using NodeJS [Express](https://expressjs.com/). Hydrogen works best with Oxygen, and any project initialized with `npm create @shopify/hydrogen@latest` will create a template that uses Oxygen. If you don't want to use Oxygen, adapting the starter template to another platform is tricky. Instead we suggest generating a new app from the [Remix CLI](https://remix.run/docs/en/1.16.1/tutorials/blog) `npx create-remix@latest` and adapting it with Hydrogen functionality.
+
+This is an example setup where we have adapted the Remix Express starter app to use Hydrogen. A few things are not yet functional:
+
+1. The app does not have a cache implementation. So sub-requests to the Storefront API won't be cachable.
```

Comment:
```suggestion
1. Because there’s no guarantees around the platform you might deploy this to, there is no cache implementation. Sub-requests to the Storefront API will not be cached. To add caching, check out options like LRU Cache or Redis.
```

Maybe we could do something like this here just to not leave folks hanging.